resume-buddy/src/main/java/com/google/sps/data/Reviewee.java

```diff
@@ -1,15 +1,32 @@
 package com.google.sps.data;
 
-/*TODO: add more information from form input into Reviewee*/
 public final class Reviewee {
 
   private final String firstName;
   private final String lastName;
   private final String email;
+  private final String school;
+  private final String year;
+  private final String career;
+  private final String degreePref;
+  private final String numYearsPref;
```

Comment:
These should probably be stored as Java enums values, since they're more like categorizations and aren't open-ended string entries.

If you want to convert the Strings to Enum values, this may help you: https://www.java67.com/2012/08/string-to-enum-in-java-conversion.html#:~:text=You%20can%20create%20Enum%20from,()%20and%20cardinal()%20methods.